java/fury-core/src/main/java/org/apache/fury/Fury.java

```diff
@@ -1380,6 +1389,10 @@ public Class<? extends Serializer> getDefaultJDKStreamSerializerType() {
     return config.getDefaultJDKStreamSerializerType();
   }
 
+  public boolean isChunkSerializeMapEnabled() {
```

Comment:
In future this will be the only way to serialize map, we'd better not make this as an option. We can add an option in AbstractMapSerializer directly, and remove that option when we support chunk based serialization in JIT mode